lib/commands/phrases.js

```diff
@@ -16,11 +15,8 @@ export default class Phrase extends Command {
   }
 
   run() {
-    let deferred = Q.defer();
-
     let index = Math.floor(Math.random() * phrasesJson.phrases.length);
-    deferred.resolve(this._buildPayload("_" + phrasesJson.phrases[index] + "_"));
 
-    return deferred.promise;
+    this.bot.send(this.payload, this._buildPayload("_" + phrasesJson.phrases[index] + "_"));
```

Comment:
Acá podés usar el backtick `this._buildPayload(``_${phrases[index]}_``)` (lo puse doble porque no hay manera de escaparlo. Maldito GH.

Reply:
👍